lib/publisher/apppublisher.go

```diff
@@ -24,7 +26,9 @@ type AppPublisher interface {
 }
 
 type AppPublisherConfig struct {
+	TenantUrl    string
 	SQSQueueName string
+	Enabled      bool
```

Comment:
who controls this enabled flag? Aren't we always publish events for apps and servers? Will this be false for windows  desktops and Kubernetes clusters?

Reply:
When I was running locally i was getting queue not found error, the flag will be enabled when we deploy on aws.